File: src/main/java/org/algorithms/leetcode/algorithms/easy/ConcatenationofArray_1929.java
package org.algorithms.leetcode.algorithms.easy;

import java.util.Arrays;

public class ConcatenationofArray_1929 {
    public static int[] getConcatenation(int[] nums) {
        int len = nums.length;
        int[] ans = new int[len * 2];

        for(int i = 0; i < len; i++){
            ans[i] = nums[i];
            ans[i+len] = nums[i];
        }
        return ans;
    }
}

File: src/Leetcode/algorithms/easy/Python/ConcatenationofArray_1929.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        """
        len(ans) = 2n
        ans[i] = nums[i]
        ans[i + n] == nums[i] 
        """
        size = len(nums) * 2
        ans = [0] * size
        for i in range(len(nums)):
            ans[i] = nums[i]
            ans[len(nums) + i] = nums[i]
        return ans